File: Android/PlayMatch/app/src/main/java/com/dam/daniel/playmatch/models/Image.java
package com.dam.daniel.playmatch.models;

public class Image {

    private int id;
    private String url;
    private int user;
    private boolean isProfile;
    private String eliminationDate;

    public Image() {}

    public Image(int id, String url, int user, boolean isProfile, String eliminationDate) {
        this.id = id;
        this.url = url;
        this.user = user;
        this.isProfile = isProfile;
        this.eliminationDate = eliminationDate;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public int getUser() {
        return user;
    }

    public void setUser(int user) {
        this.user = user;
    }

    public boolean isProfile() {
        return isProfile;
    }

    public void setProfile(boolean profile) {
        isProfile = profile;
    }

    public String getEliminationDate() {
        return eliminationDate;
    }

    public void setEliminationDate(String eliminationDate) {
        this.eliminationDate = eliminationDate;
    }

    @Override
    public String toString() {
        return "Image{" +
                "id=" + id +
                ", url='" + url + '\'' +
                ", user=" + user +
                ", isProfile=" + isProfile +
                ", eliminationDate=" + eliminationDate +
                '}';
    }
}

File: Backend/PlayMatch/src/models/Image.ts
export class Image{

    url: string;
    user: number;
	isProfile: boolean;
	eliminationDate: string;

    constructor(url?: string, user?: number, isProfile?: boolean, eliminationDate?: string){
        this.url = url;
        this.user = user;
		this.isProfile = isProfile;
		this.eliminationDate = eliminationDate;
	}
	
	public getUrl()
	{
		return this.url;
	}

	public setUrl(url: string)
	{
		this.url = url;
	}

	public getUser()
	{
		return this.user;
	}

	public setUser(user: number)
	{
		this.user = user;
	}

	public getIsprofile()
	{
		return this.isProfile;
	}

	public setIsprofile(isProfile: boolean)
	{
		this.isProfile = isProfile;
	}

	public getEliminationDate()
	{
		return this.eliminationDate;
	}

	public setEliminationDate(eliminationDate: string)
	{
		this.eliminationDate = eliminationDate;
	}

}